dom/app.js

```diff
@@ -0,0 +1,30 @@
+const btnCreate = document.querySelector('.btn-main');
+const btnToggle = document.querySelector('.btn-toggle');
+const btnRemove = document.querySelector('.btn-remove');
+
+btnCreate.addEventListener('click', () => {
+    const input = document.querySelector('.input-main');
+    const list = document.querySelector('ul');
+
+    list.insertAdjacentHTML(
+        'afterbegin',
+        `<li>${input.value}</li>`
+    );
+    input.value = '';
+});
+
+btnRemove.addEventListener('click', () => {
+    const lastItem = document.querySelector('li:last-child');
+    lastItem.remove();
```

Comment:
Can you inline this and remove the use of the variable?
`document.querySelector('li:last-child').remove();`

Reply:
Done.